renderdoc/os/posix/android/android_stringio.cpp

```diff
@@ -68,7 +68,7 @@ rdcstr GetTempRootPath()
   // This is the same as returned by getExternalFilesDir(). It might possibly change in the future.
   rdcstr package;
   GetExecutableFilename(package);
-  return "/sdcard/Android/data/" + package + "/files";
+  return "/sdcard/Android/media/" + package + "/files";
```

Comment:
I'd like to continue to use the old path for all Android versions up to Android 11, and only use the `media` path for Android 11. That way we can be more confident this change won't break on an older version of Android.

Reply:
I've been trying to find a clean way of doing this - Is there any means currently with which I can access the current Remote Host (even better - the AndroidRemoteServer instance) from the android_stringio.cpp file? Like maybe a static variable somewhere for the current Remote Host state? I haven't been able to find an easy way to do this without modifying the os_specifics.h file to require passing the deviceID down so that I can query the Android version of the device currently connected

Reply:
This is the Android specific implementation of `GetTempRootPath()` so it's executing on the device itself. The host side `GetTempRootPath()` will be returning the path for local captures which isn't affected by any Android changes. The query in this function will need to be some android-specific check to get the version running.

I think anywhere on the host side here we want to switch between paths should already have the deviceID available to do a query.

Reply:
Ohhhh I see, thank you for the clarification! I should be good to go then :)